### ISSUE TITLE
[Feat]: OAuth scope doesn't replace default commas with spaces

### ISSUE THREAD

**Issue by lanedsmu** [lowcoder-org/lowcoder#915]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

In the setup wizard for generic OAuth authentication, the default scope looks like this:
![image](https://github.com/lowcoder-org/lowcoder/assets/80843079/ab0034fb-b32c-446e-8226-14aa53f0e2a9)

Oauth specs call for the scopes to be separated by spaces, and when this default is left in place, we get the following response from  the auth provider:
```
GET
    https://<lowcoderURL>/user/auth/oauth/redirect?error=invalid_client&error_description=AADSTS650053: The application 'OIDCName' asked for scope 'openid,profile,email,offline_access' that doesn't exist on the resource '00000003-0000-0000-c000-000000000000'. Contact the app vendor. Trace ID: 31462217-5a92-430c-8ec7-1c2b20aea000 Correlation ID: 889b6cd1-3ac2-45ae-8f02-32a523f7fab4 Timestamp: 2024-05-28 18:26:48Z&state=f81bc204
```

It appears the process to replace the comma-separated list with spaces isn't working as intended.

### Expected Behavior

Scope is sent to the provider as:
```openid profile email offline_access```

### Steps to reproduce

Configure a generic oauth provider, keeping the scope as provided by default.  Attempt an authentication session, and the response in the dev console will include the above error.

### Environment

docker multi-image deployment on enterprise linux

**Comment by FalkWolsky** [comment]
We changed the type to Enhancement, since this is not a bug technically. 
The input field does the job in general, just the user would have to know about the space (which is the standard in OAUTH Scopes). What we will do anyhow, we will replace "," and ";" at this field automatically by the space - so we help the user for a better user experience.

**Comment by raheeliftikhar5** [comment]
@lanedsmu this is resolved in 2.4.1
